packages/ui/src/RSelect/index.tsx

```diff
@@ -64,83 +54,62 @@ export function SelectItem(props) {
     }
   };
 
-  const options = useMemo(() => {
+  const menuItems = useMemo(() => {
     if (!Array.isArray(data)) return [];
-    return data
-      .map((item, index) =>
-        extractOptionAttributes(item, index, textKey, valueKey),
-      )
-      .map(({ text, value, index }) => (
-        <Select.Item key={`${text} + ${value}`} index={index} value={value}>
-          <Select.ItemText>
-            {text.charAt(0).toUpperCase() + text.slice(1)}
-          </Select.ItemText>
-          <Select.ItemIndicator marginLeft="auto">
-            <Check size={16} />
-          </Select.ItemIndicator>
-        </Select.Item>
-      ));
+    return data.map((item, index) => {
+      const { text, value, index: itemIndex } = extractOptionAttributes(item, index, textKey, valueKey);
+      return {
+        label: text.charAt(0).toUpperCase() + text.slice(1),
+        onSelect: () => handleChange(value),
+      };
+    });
   }, [data, textKey, valueKey]);
 
-  // Conditional rendering based on options
-  if (options.length === 0) {
-    return <Text>No options available</Text>;
-  }
-
   return (
     <Select
       value={value}
       disablePreventBodyScroll
       onValueChange={handleChange}
       {...forwardedProps}
     >
-      <Select.Trigger width={220} iconAfter={ChevronDown}>
```

Comment:
Nice job but I think we should support both via prop is my thinking

just select with prop of native = sheet
Select with prop native + zeego = zeego dropdown

Reply:
@andrew-bierman Could you please review the code? If it is correct, I will apply it to the other instances where we use dropdown menus.

Reply:
yes this looks good, nice job :)